src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -281,7 +306,7 @@
    :component-did-mount
    (fn [{:keys [state props refs this]}]
      (endpoints/call-ajax-orch
-       {:endpoint (endpoints/get-workspace-method-config (:workspace-id props) (:config props))
+       {:endpoint (endpoints/get-validated-workspace-method-config (:workspace-id props) (:config props))
```

Comment:
I think this is a bit different than I am used to, but I like it. Keep it as you have it.